clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -1093,9 +1106,157 @@
     );
 
     public static final Schema[] INIT_PRODUCER_ID_REQUEST = new Schema[] {INIT_PRODUCER_ID_REQUEST_V0};
-
     public static final Schema[] INIT_PRODUCER_ID_RESPONSE = new Schema[] {INIT_PRODUCER_ID_RESPONSE_V0};
 
+    public static final Schema ADD_PARTITIONS_TO_TXN_REQUEST_V0 = new Schema(
+            new Field("transactional_id",
+                    STRING,
+                    "The transactional id corresponding to the transaction."),
+            new Field("pid",
+                    INT64,
+                    "Current PID in use by the transactional id"),
```

Comment:
I treat PID as an acronym (producer ID). When used in a sentence, I think it should be capitalized like other acronyms (e.g. SSL). In code, however, I think we can treat it as a word, so we have `InitPidRequest` instead of `InitPIDRequest`. This may seem odd, but it is conventional (e.g. see `SslChannelBuilder` or `Crc32`).